src/document/DocumentCommandHandlers.js

```diff
@@ -583,15 +583,22 @@ define(function (require, exports, module) {
      *   the given base name
      */
     function _getUntitledFileSuggestion(dir, fileNameOptions, isFolder) {
-        var baseFileName = fileNameOptions.baseFileName,
-            extension    = fileNameOptions.extension ? fileNameOptions.extension.replace(/^\.?/, ".") : "";
+        var baseFileName  = fileNameOptions.baseFileName,
+            extension     = fileNameOptions.extension ? fileNameOptions.extension.replace(/^\.?/, ".") : "",
+            suggestedName = "";
```

Comment:
Don't bother setting this to the empty string, because all code paths below set it to a new string as needed.